packages/interface/src/utils/parseRange.test.ts

```diff
@@ -0,0 +1,61 @@
+/**
+ * Original tests from https://github.com/euank/node-parse-numeric-range
+ */
+
+import { describe, expect, it } from 'vitest';
+import { parseRange } from './parseRange';
+
+function peq(x: string, y: number[]) {
+    expect(parseRange(x)).to.eql(y);
+}
+
+describe('parseRange', function () {
+    it('should parse 1', function () {
+        peq('1', [1]);
+    });
+    it('should parse 1,1', function () {
+        peq('1,1', [1, 1]);
+    });
+    it('should parse 1-5', function () {
+        peq('1-5', [1, 2, 3, 4, 5]);
+    });
+    it('should parse 5-1', function () {
+        peq('5-1', [5, 4, 3, 2, 1]);
+    });
+    it('should parse 1-3,5-6', function () {
+        peq('1-3,5-6', [1, 2, 3, 5, 6]);
+    });
+    it('should parse 10..15', function () {
+        peq('10..15', [10, 11, 12, 13, 14, 15]);
+    });
+    it('should parse 10...15', function () {
+        peq('10...15', [10, 11, 12, 13, 14]);
+    });
+    it('should parse 10..12,13...15,2,8', function () {
+        peq('10..12,13...15,2,8', [10, 11, 12, 13, 14, 2, 8]);
+    });
+    it('should parse ""', function () {
+        peq('', []);
+    });
+    it('should parse -5', function () {
+        peq('-5', [-5]);
```

Comment:
I would expect `-5` to mean `<start value>-5`

Reply:
Yes, now fixed in last commit.